src/main/java/Scheduler.java

```diff
@@ -0,0 +1,22 @@
+import org.quartz.*;
+import org.quartz.impl.StdSchedulerFactory;
+import java.io.IOException;
+
+public class Scheduler {
+
+    public static void main(String[] args) throws IOException, SchedulerException {
+
+        JobDetail job = JobBuilder.newJob(CreateCRE.class).build();
+
+        Trigger trigger = TriggerBuilder.newTrigger().withIdentity("getTrigger").withSchedule(SimpleScheduleBuilder
+                .simpleSchedule().withIntervalInHours(24).repeatForever()).build();
+
+
+        org.quartz.Scheduler scheduler = StdSchedulerFactory.getDefaultScheduler();
+
+        scheduler.start();
+        scheduler.scheduleJob(job, trigger);
```

Comment:
Hmm, I know it's copy/paste from mkyong tutorial but try to switch those 2 instruction and check if works (first schedule than start - I think it's more natural). If it doesn't work leave it like this.

scheduler.scheduleJob(job, trigger);
scheduler.start();